pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go

```diff
@@ -683,13 +738,107 @@ func TestPodTopologySpreadScore(t *testing.T) {
 				{Name: "node-b", Score: 0},
 			},
 		},
+		{
+			// +------------------------+------------------------+
+			// |         "zone1"        |          ""            |
+			// +------------------------+------------------------+
+			// | node-a |    node-b     | node-x |     node-y    |
+			// +--------+---------------+--------+---------------+
+			// | P{foo} | P{bar} P{bar} |        | P{foo} P{bar} |
+			// +--------+---------------+--------+---------------+
+			// The matching pods spread as 0/2/0/1
+			name: "default constraints on node",
+			pod:  st.MakePod().Name("p").Namespace("default").Label("bar", "").OwnerReference("rs1", appsv1.SchemeGroupVersion.WithKind("ReplicaSet")).Obj(),
+			existingPods: []*v1.Pod{
+				st.MakePod().Namespace("default").Name("p-a1").Node("node-a").Label("foo", "").Obj(),
```

Comment:
I would remove the "foo" pods, as they are distracting for the purpose of this test.